bin/dephpugger

```diff
@@ -1,89 +1,32 @@
 #!/usr/bin/env php
 <?php
 
-$autoloads = [
-    dirname(dirname(dirname(dirname(__DIR__)))) . DIRECTORY_SEPARATOR,
-    
+$availableAutoloadPaths = [
+    __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'autoload.php',
+    __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php',
+    __DIR__ . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php',
 ];
 
-// Checking if file is found
-$dirPath = false;
-foreach($autoloads as $path) {
-    $autoload = $path . 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php';
-    if(file_exists($autoload)) {
-        require_once $autoload;
-        $dirPath = $path;
+foreach ($availableAutoloadPaths as  $availableAutoloadPath) {
```

Comment:
Could you change the second variable name, please?
Because `$availableAutoloadPaths` and `$availableAutoloadPath` are very similar.

My suggestion is `$currentAvailableAutoloadPath`, but if you prefer another name, no problem ;D